libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -325,6 +325,16 @@ class AppHeaderCore extends Component<IAppHeaderProps & WrappedComponentProps, I
                     className="gd-header-menu-horizontal"
                 />
 
+                {!!this.props.isDisplayUpsellButton && (
```

Comment:
`!!` operator to convert the type to boolean, but `isDisplayUpsellButton` has type is boolean, so we don't need !! . :)

Reply:
I changed it thanks